cmds/dump.c

```diff
@@ -533,6 +530,13 @@ static void dump_raw_header(struct uftrace_dump_ops *ops, struct uftrace_data *h
 	int i;
 	char buf[1024];
 	struct uftrace_raw_dump *raw = container_of(ops, typeof(*raw), ops);
+	const char *feat_str[] = { "PLTHOOK",	 "TASK_SESSION", "KERNEL",     "ARGUMENT",
+				   "RETVAL",	 "SYM_REL_ADDR", "MAX_STACK",  "EVENT",
+				   "PERF_EVENT", "AUTO_ARGS",	 "DEBUG_INFO", "ESTIMATE_RETURN" };
+	const char *info_str[] = { "EXE_NAME",	   "EXE_BUILD_ID", "EXIT_STATUS", "CMDLINE",
+				   "CPUINFO",	   "MEMINFO",	   "OSINFO",	  "TASKINFO",
+				   "USAGEINFO",	   "LOADINFO",	   "ARG_SPEC",	  "RECORD_DATE",
+				   "PATTERN_TYPE", "VERSION" };
```

Comment:
While it was not there originally, please put a comma after the last element so that it can format the code 'properly'.